api/app/settings/common.py

```diff
@@ -942,3 +942,81 @@
 FLAGSMITH_ON_FLAGSMITH_SERVER_API_URL = env(
     "FLAGSMITH_ON_FLAGSMITH_SERVER_API_URL", default=FLAGSMITH_ON_FLAGSMITH_API_URL
 )
+
+# LDAP setting
+LDAP_INSTALLED = importlib.util.find_spec("ldap")
+if LDAP_INSTALLED:
+    # The URL of the LDAP server.
+    LDAP_AUTH_URL = env.str("LDAP_AUTH_URL", None)
+    if LDAP_AUTH_URL:
+        AUTHENTICATION_BACKENDS.insert(0, "django_python3_ldap.auth.LDAPBackend")
```

Comment:
Should this throw an exception if a URL is not set?

Reply:
No, because we will ship every private cloud image with LDAP, and it's up to the client if they want to enable it or not. is that correct @matthewelwell